PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+# Description of feature
+...
+
+# Checklist
+- [ ] (if applicable) added new ENV variables to the readme
```

Comment:
(If applicable) added example values of new ENV variables to .env.local and explained them in README.md?